common/config/package.json

```diff
@@ -18,7 +18,7 @@
     "@octokit/rest": "~18.0.6",
     "@rollup/plugin-commonjs": "~17.1.0",
     "@types/node": "^14.14.10",
-    "beachball": "~2.20.0",
+    "beachball": "2.20.0",
```

Comment:
Ensure to lock the beachball version in order to keep beachball-changelog-only.js
Which is a work around before https://github.com/microsoft/beachball/issues/634 gets resolved